src/pudl/metadata/fields.py

```diff
@@ -351,6 +351,27 @@
     "ferc_account_description": {
         "type": "string",
     },
+    "fuel_agg": {
+        "type": "string",
+        "description": "Category of fuel aggregation in EIA bulk electricity data.",
+    },
+    "geo_agg": {
+        "type": "string",
+        "description": "Category of geographic aggregation in EIA bulk electricity data.",
+    },
+    "sector_agg": {
+        "type": "string",
+        "description": "Category of sectoral aggregation in EIA bulk electricity data.",
+    },
+    "temporal_agg": {
+        "type": "string",
+        "description": "Category of temporal aggregation in EIA bulk electricity data.",
+    },
+    "receipts_mmbtu": {
+        "type": "number",
+        "description": "Category of fuel aggregation in EIA bulk electricity data.",
+        "unit": "MMBtu",
+    },
```

Comment:
oops, that description was a copy/paste error.

For naming consistency, I checked the sqlite table `fuel_receipts_costs_eia923`; I forgot that we have more names in the python output functions. I don't think there is a technical difference between receipts/received, except that "receipts" is more consistent with EIA terminology that many users are probably familiar with. But because we already use "received" in the rest of PUDL then I'll use that. I'll change this to `fuel_received_mmbtu`